docs/sources/articles/dockerfile_best-practices.md

```diff
@@ -335,12 +331,31 @@ fi
 exec "$@"
 ```
 
-That script then gets copied into the container and run via `ENTRYPOINT` on
-container startup:
+> **Note**: This script uses [the `exec` Bash command](http://wiki.bash-hackers.org/commands/builtin/exec)
+> so that the final running application becomes the container's `PID 1` so that
+> it will receive any Unix signals sent to the container. See the
```

Comment:
I don't like the "so that" *twice* in this sentence. Also "This script" and "it will receive" may need to be more specific, it's a bit hard to follow what they refer to.